package.json

```diff
@@ -54,9 +54,15 @@
     }
   },
   "devDependencies": {
+    "babel-jest": "^23.0.1",
```

Comment:
This was required for me to run `npm test`.

Reply:
We already added `babel-jest` in `rc-tools`.